docs/javascripts/compatibility_list.js

```diff
@@ -22,7 +22,98 @@
  SOFTWARE.
```

Comment:
This script is mostly yours now. You may relicense and take credit for it in the header as you choose, @Razzile.